src/components/editTestCase/qdm/LeftPanel/ElementsTab/Elements/DataElementsCard/attributes/DisplayAttributeInputs.tsx

```diff
@@ -262,6 +263,14 @@ const DisplayAttributeInputs = ({
             onInputAdd={onInputAdd}
           />
         );
+      case "FacilityLocation":
+        return (
+          <FacilityLocation
+            canEdit={true}
```

Comment:
Recently we completed the work on handling Edit privileges for Editing QDM testcases. Even though all inputs in this  DisplayAttributeInputs has a canEdit field, it is not required to pass in when using it from DisplayAttributeInputs, especially if we had to hard code it. 

May be we can remove it, make it optional in "FacilityLocation". Looks like we just don't need to worry about canEdit in DisplayAttributeInputs.tsx

Reply:
I just saw that canEdit is chained as a required prop for its children Components. Damn it !!  I guess we can leave it for now then.

Reply:
I'm not sure what was our plan with canEdit. all of the components used here have hardcoded canedit made me think that we are going to address in some other story. if that's not required then I'll just remove it. thanks for pointing it.

Reply:
yeah, it is chained. I made it optional with default true.